packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1172,6 +1172,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // The last message processed at the time of summary. If there are no messages, nothing has changed from
```

Comment:
It would be great to have some subset (or maybe all) of these properties to be logged in telemetry when we open a file. Without that, the usefulness of this data (you are adding) is very minimal, as usually we do not have access to files, only to telemetry